client/index.html

```diff
@@ -261,14 +267,14 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
-									<% themes.forEach(function(themeName) { %>
-										<option value="<%= themeName %>">
-											<%= themeName.charAt(0).toUpperCase() + themeName.slice(1) %>
+									{{#each themes}}
+										<option value="{{this}}">
+											{{firstLetterUppercase this}}
```

Comment:
I moved this into the server and just did the simple structure. I accept that we could easily bikeshed, so please ... unless there's a very good reason don't spend time arguing over the structure. This way we don't have to rename the helper function (because, you know, that's a big bikeshed as well) and it has simplified other things as well. We can think about a better structure when we get the theme manifest done sometime down the line.